packages/app-bridge/src/types/PlatformAppContext.ts

```diff
@@ -0,0 +1,15 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```

Comment:
What is this file?
Data like context/state must be in AppBridgePlatformApp

Reply:
files moved